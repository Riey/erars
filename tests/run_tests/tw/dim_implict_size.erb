@SYSTEM_TITLE
#DIMS FOO = "1", "2", "3"
PRINTFORML FOO:2 = %FOO:2%
