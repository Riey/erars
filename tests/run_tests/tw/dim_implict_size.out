FOO:2 = 3
